net.openhabittracker.app.metainfo.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="utf-8"?>
+<component type="desktop">
+  <id>net.openhabittracker.app</id>
```

Comment:
`app` isn't an application name. The last component is supposed to be the application name.